### PR TITLE
Fix signer & request builder, and add new api

### PR DESCRIPTION
**Pull request by qychrisyang**

### CODE REVIEW

File: QingStorSDK.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -355,7 +352,7 @@
 			isa = PBXProject;
 			attributes = {
 				LastSwiftUpdateCheck = 0810;
```
Comment (Xuanwo) [inline QingStorSDK.xcodeproj/project.pbxproj:354]: LastSwiftUpdateCheck don't need to update?

File: Sources/Info.plist [hunk 2]
```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>2.0.0-beta.1</string>
+	<string>2.2</string>
```
Comment (Xuanwo) [inline Sources/Info.plist]: not 2.2.0 ?

File: Sources/RequestBuilder.swift [hunk 3]
```diff
@@ -156,14 +156,14 @@ open class RequestBuilder {
                 }()
 
                 return {
-                    var result = "\(QingStorSDKVersion) \(osNameVersion) "
+                    var result = "\(QingStorSDKVersion) (\(osNameVersion))"
 
                     if let executable = executable, let appVersion = appVersion {
-                        result += "\(executable)/\(appVersion) "
+                        result += " \(executable)/\(appVersion)"
```
Comment (Xuanwo) [inline Sources/RequestBuilder.swift:162]: a space?

File: QingStorSDK.xcodeproj/project.pbxproj (hunk 1 above)
Comment (qychrisyang) [reply QingStorSDK.xcodeproj/project.pbxproj:354]: This code is automatically generated, I think it should not be manually changed.

File: Sources/Info.plist (hunk 2 above)
Comment (qychrisyang) [reply Sources/Info.plist]: Should be 2.2.0, I will correct it

File: Sources/RequestBuilder.swift (hunk 3 above)
Comment (qychrisyang) [reply Sources/RequestBuilder.swift:162]: is a space.

**Review by Xuanwo — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,14 @@
 # Change Log
 All notable changes to QingStor SDK for Swift will be documented in this file.
 
+## [v2.2.0] - 2016-03-11
+
+### Fixed
+- Fix signer & request builder bug.
+
```
Comment (Xuanwo) [inline CHANGELOG.md:8]: query_sign should be listed if fixed.

Comment (qychrisyang) [reply CHANGELOG.md:8]: I think it also signer bug.

Comment (Xuanwo) [reply CHANGELOG.md:8]: OK, you can merge now.

**Pull request merged**